src/styles/index.css

```diff
@@ -0,0 +1,275 @@
+.body {
+  color: #060b35;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+  position: relative;
+  margin-left: 20px;
+  height: 60px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  color: #060b35;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 60px;
+  font-weight: 500;
+  text-decoration: none;
+  white-space: nowrap;
+  display: block;
+  transition-duration: 300ms;
+}
+
+.nav__link--is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link--is-active::after {
+  content: '';
+  display: block;
+  background-color: #00acdc;
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+}
+
+.card__title {
+  padding-inline: 16px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  position: absolute;
+  top: 206px;
+  transition-duration: 300ms;
+}
+
+.card__title:hover {
+  color: #34568b;
+}
+
+.card__product-code {
+  padding-inline: 16px;
+  position: absolute;
+  top: 246px;
+  color: #616070;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+}
+
+.card__price {
+  padding-inline: 16px;
+  width: 166px;
+  position: absolute;
+  top: 316px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-text {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-price {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__button {
+  padding-inline: 16px;
+  display: flex;
+}
+
+.card__buy {
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-align: center;
+  transition-duration: 300ms;
+  position: absolute;
+  top: 350px;
+  text-transform: uppercase;
+  background-color: #00acdc;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  align-content: center;
+}
+
+.card__buy:hover {
+  cursor: pointer;
+  box-sizing: border-box;
+  border-radius: 5px;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+  background-color: #fff;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
```

Comment:
Consider using relative units (like percentages or ems) for padding to improve responsiveness.